src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.search-bar {
+  margin: 20px 9px 0;
+}
+
+.search-bar__container {
+  padding: 22px 25px;
+  color: #3d4e61;
```

Comment:
let's move this color to some var and reuse it everywhere

Reply:
Done)